src/workerd/io/compatibility-date.capnp

```diff
@@ -1061,4 +1061,11 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
     $impliedByAfterDate(name = "nodeJsCompat", date = "2025-10-01");
   # Enables the Node.js non-functional stub vm module. It is required to use this flag with
   # nodejs_compat (or nodejs_compat_v2).
+
+  enableNodeJsV8Module @122 :Bool
+    $compatEnableFlag("enable_nodejs_v8_module")
+    $compatDisableFlag("disable_nodejs_v8_module")
+    $impliedByAfterDate(name = "nodeJsCompat", date = "2025-10-01");
```

Comment:
I don't think 2025-10-01 is realistic.

Because we already have a few modules in the workers-sdk queue.

Also I think it should be experimental until we do some testing.

Reply:
Marked experimental and pushed the date out to 2025-10-15